explainer.md

```diff
@@ -316,6 +354,27 @@ try {
 }
 ```
 
+### Delete a partitioned cookie
+
+If the user agent supports [cookie partitioning](https://github.com/WICG/CHIPS)
+then it is possible for a site to set both a partitioned and unpartitioned
+cookie with the same name.
+
+In this edge case, if a site would like to distinguish between whether they
+want to delete their partitioned and unpartitioned cookie, they can provide
+a `partitioned` attribute. If the site wants to delete the partitioned cookie,
+the site could use:
+
+```javascript
+await cookieStore.delete({
+  name: '__Host-third_party_session_id',
+  partitioned: true
+});
+```
+
+If the site wants to delete the unpartitioned cookie, change the `partitioned`
+field to `false`. If the field is not present, the value defaults to `false`.
```

Comment:
In the IDL the default isn't given for `CookieStoreDeleteOptions`

Reply:
Added default to IDL